src/pages/Home.tsx

```diff
@@ -36,7 +36,7 @@ export default function Home() {
     <PluginPage
       pageNav={{
         text: 'Advisor',
-        subTitle: 'Keep Grafana running smoothly and securely',
+        subTitle: 'Advisor runs regular health checks on your Grafana instance, data sources and plugins and suggests improvement action items',
```

Comment:
> regular health checks

It can also run checks on-demand, we could optionally add this detail in. 

> on your Grafana instance, data sources and plugins

Maybe i'm overthinking this, but this makes it sound like they are peer entities, rather than the data source instances and the plugins being part of the grafana instance. 

> suggests improvement action items

How about "suggests actions to fix identified issues" or something along those lines